.buildkite/ftr_configs.yml

```diff
@@ -321,6 +321,8 @@ enabled:
   - x-pack/test/functional/apps/saved_query_management/config.ts
   - x-pack/test/functional/apps/security/config.ts
   - x-pack/test/functional/apps/slo/embeddables/config.ts
+  - x-pack/test/functional/apps/search_playground/config.ts
+  - x-pack/test/functional/apps/search_playground/serverless.config.ts
```

Comment:
I don't think this should be added here, I'm not positive you should create this config at all. The serverless tests should get imported into the existing [search suite](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/test_suites/search/index.ts)

Reply:
<img width="1414" alt="image" src="https://github.com/elastic/kibana/assets/17390745/c046c6d4-4d58-465a-a07c-770f210bea75">
there is a bunch of logic like this and even more